narratives/ncov_sit-rep_2020-04-17.md

```diff
@@ -0,0 +1,238 @@
+<br><br>
+The New York clade is both flanked by and intercalated with sequences from Europe, which strongly suggests that most early cases resulted from introductions from Europe.
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 5-->
+# [Many introductions to New York](https://nextstrain.org/ncov/north-america/2020-04-17?label=clade:A2&c=division&d=tree,map&f_division=Europe,New%20York,Asia&p=grid&r=division)
+
+Although the majority of the New York outbreak was seeded by European introductions, by adding in more contextual sequences, we get a more nuanced view.
+True to its cosmopolitan nature, there are also cases in New York that are closely related to samples from every other region of the world.
+<br><br>
+Notably, although the early New York cases were most likely represent inbound transmission, the directionality of these transmissions is less clear later on in the outbreak. As we reported on [last week](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-10?n=11), there is also evidence for recent reintroductions from the U.S. to Europe and Asia.
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 6-->
+# [Intermingling of cases from all 5 NYC boroughs](https://nextstrain.org/ncov/north-america/2020-04-17?c=location&d=tree,map&f_division=New%20York&f_location=Bronx,Brooklyn,Queens,Staten%20Island,Manhattan&label=clade:A2&p=full&r=location)
+
+Within New York City, we see little segregation of cases by borough. This is consistent with frequent mixing of cases across the city.
+<!-- This is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 7-->
+# [California outbreak related to both nearby and distant states'](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&f_division=California,New%20York,Washington&r=location&d=tree,map)
+
+We also see evidence of multiple introductions between states. For example, cases from California span the genetic diversity of the tree, grouping together with samples from both nearby Washington State and New York. This is evidence for transmission between California and both nearby and distant states.
+
+<!-- This is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 8-->
+# [Midwest outbreaks were seeded by many introductions](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_country=USA&f_division=Illinois,Indiana,Iowa,Kansas,Michigan,Minnesota,Nebraska,Ohio,Wisconsin&p=full&r=division)
+
+Turning our attention to the Midwestern United States, we see that cases from this area are spread evenly across the tree, rather than forming a series of tight clusters. This tells us that there have been many independent introductions to these states, with several different transmission chains circulating in parallel.
```

Comment:
```suggestion
Turning our attention to the Midwestern United States, we see that cases from this area are spread evenly across the tree, rather than forming a series of tight clusters. This tells us that there have been many independent introductions to these states, with several different transmission chains circulating simultaneously.
```